src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -151,9 +151,13 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
 
         auto rep_file_name = ip->getAlphaFieldValue(fields, objectSchemaProps, "representation_file_name");
         fs::path rep_file_path = DataSystemVariables::CheckForActualFilePath(state, fs::path(rep_file_name), std::string(RoutineName));
+        if (rep_file_path.empty()) {
+            ErrorsFound = true;
+        }
         std::pair<EnergyPlusData *, std::string> callbackPair{&state,
                                                               format("{} \"{}\"", state.dataIPShortCut->cCurrentModuleObject, thisObjectName)};
         tk205::set_error_handler(tk205ErrCallback, &callbackPair);
+        Btwxt::LOG_LEVEL = static_cast<int>(Btwxt::MsgLevel::MSG_WARN);
```

Comment:
Feels a bit odd to have to static cast when both sides are Btwxt expressions, but that's not my concern here.

Reply:
Yeah, this is not great. It's something we will address in a future version of Btwxt.